logcat_pinger.sh

```diff
@@ -7,14 +7,16 @@ export TEST_RUNNING_FILE="$(mktemp)"
 # Args:
 # $1 host_to_ping
 function ping_to_logcat {
+  echo "ping_to_logcat TEST_RUNNING_FILE=$TEST_RUNNING_FILE"
   MINIMUM_LOOP_TIME=5
-  PINGS_IN_BLOCK=10
+  PINGS_IN_BLOCK=4
+  INTERVAL=3
   touch "${TEST_RUNNING_FILE:?}"
   (
     while [ -f "${TEST_RUNNING_FILE:?}" ]
     do
       date_at_start=$(date +%s)
-      adb shell "ping -c${PINGS_IN_BLOCK} $1 |while read line ; do log -pd -tQApinger \$line ; done"
+      adb shell "ping -c${PINGS_IN_BLOCK} -i${INTERVAL} $1 |while read line  ; do line=\$(echo \$line | sed -e 's/-/_/g') ; log -pd -tQApinger \"\$line .\" ; done"
```

Comment:
it must be oneline currently as it is run on the device, not on the host computer. 

But actually I was planning of making it into a script that is uploaded to the device, if that works.